milli/src/update/index_documents/transform.rs

```diff
@@ -230,38 +244,40 @@ impl Transform<'_, '_> {
             documents_seen: documents_count,
         });
 
+        self.index.put_fields_ids_map(wtxn, &fields_ids_map)?;
+        self.index.put_primary_key(wtxn, &primary_key_name)?;
```

Comment:
Ok, now the transform struct writes into the database, to keep a constant state? For example the new fields it found when iterating through documents?

Reply:
that's right